build.sbt

```diff
@@ -148,6 +152,9 @@ lazy val allOld = (project in file("allOld"))
           case modulePostgresEx(e) =>
             e.copy(child = e.child :+ exclude(Dependencies.modulePostgres.value))
 
+          case moduleOracleEx(e) =>
```

Comment:
You don't have to add a new module in the `allOld` module. `allOld` is only for backward compatibility. Just add `moduleOracle` into the `root` project and it would be enough.